src/main/java/com/vinylteam/vinyl/service/impl/DefaultPasswordRecoveryService.java

```diff
@@ -23,88 +23,106 @@
 @Slf4j
 @Service
 @RequiredArgsConstructor
-@PropertySource("classpath:application.properties")
 public class DefaultPasswordRecoveryService implements PasswordRecoveryService {
     private static final String RECOVERY_MESSAGE = "Hello, to change your password, follow this link: \n{applicationLink}/recoveryPassword/newPassword?token=";
-
-    private final PasswordRecoveryDao passwordRecoveryDao;
-    private final UserService userService;
-    private final MailSender mailSender;
     @Value("${application.link}")
     private String applicationLink;
     @Value("${recoveryToken.live.hours}")
```

Comment:
You could use static constants for such declarations
Actually the point of @value is to have its actual value specified in a property file
You could also use : operator inside @value so that default value will be used if specified key does not exist in a property file

@Value("${recoveryToken.live.hours:default-value}")

Reply:
The default value (24) is specified for the case when there is no appropriate application property.